packages/material-ui/src/Grid/Grid.js

```diff
@@ -173,25 +173,25 @@ export const styles = (theme) => ({
   'align-content-xs-space-around': {
     alignContent: 'space-around',
   },
-  /* Styles applied to the root element if `justify="center"`. */
-  'justify-xs-center': {
-    justifyContent: 'center',
+  /* Styles applied to the root element if `justifyContent="center"`. */
+  'justify-content-xs-center': {
+    justifyContentContent: 'center',
   },
-  /* Styles applied to the root element if `justify="flex-end"`. */
-  'justify-xs-flex-end': {
-    justifyContent: 'flex-end',
+  /* Styles applied to the root element if `justifyContent="flex-end"`. */
+  'justify-content-xs-flex-end': {
+    justifyContentContent: 'flex-end',
```

Comment:
This explains why Argos fails

```suggestion
    justifyContent: 'flex-end',
```

Reply:
Oops